json/Item_Stack_Music.txt

```diff
@@ -2902,14 +2902,14 @@
 		"jp_text": "PSO2 イベント「コミカル」",
 		"tr_text": "PSO2 Event [Comical]",
 		"jp_explain": "ジュークボックス用音楽再生ディスク。\nＰＳＯ２ ＃１４２\n『Comedy Again』",
-		"tr_explain": "By using this ticket, you will\nunlock the new lobby action\n\"PSO2 Event [Comical]\" for all\ncharacters on your account."
+		"tr_explain": ""
 	},
 	{
 		"assign": "3190517",
```

Comment:
As well?

Reply:
This isn't a lobby action, it's a music disk, thus that description was rightfully removed.